app/src/main/java/com/group80/uoftinder/RecommendationInteractor.java

```diff
@@ -0,0 +1,82 @@
+package com.group80.uoftinder;
+
+import com.group80.uoftinder.entities.User;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+public class RecommendationInteractor {
+
+//    private User currentUser;
+    private List<User> compatibilityList;
```

Comment:
I don't know if I'm missing something but where exactly does this class get the compatibility list. It seems that you've initialized the compatibilityList without actually giving it a value anywhere? I presume that this will be later initialized in the constructor as it is empty at the moment but I just want to make sure. Also, I want to make a note to myself that we need to discuss how to store the users in the compatibilityList. Should it be a list of User objects or a list of strings (Uids). It is probably better to have it a list of User objects since most of our methods and classes require the actual user object but we should probably talk about this and have a concrete plan.

Reply:
Thanks Nicholas! I'm waiting on your PR to be merged to add getters and setters into your class to grab compatibility list. If you scroll down further in this same class, there are two methods commented out. They correspond to what I briefly messaged you offline about regarding using one method to turn a list of String Uids into a list of User objects. It might be more efficient for us to chat the details of integrating these offline.